src/api.js

```diff
@@ -259,7 +259,7 @@ class VLWholeSlideMicroscopyImageViewer {
     this[_client] = options.client;
 
     if (!('retrieveRendered' in options)) {
-      options.retrieveRendered = false;
+      options.retrieveRendered = true;
```

Comment:
This is logically incorrect. It should be

```js
if (!('retrieveRendered' in options)) {
  options.retrieveRendered = false;
}
```

Reply:
I think that was from your commit: https://github.com/dcmjs-org/dicom-microscopy-viewer/pull/29/commits/a4e25a014b37225558ac6e5e1a0c591b1c2f55d0

Currently in Master, if `options` has no property `retrieveRendered`, it will be set to false, and `retrieveRendered` will not be used.

This commit would change it so that if the `retrieveRendered` property is not set in options, it would be set to true (i.e. use retrieveRendered by default). I remember we spoke about doing this a while ago, but the change has never made it into master - `retrieveRendered` is not currently the default.

I switched the examples themselves because they are hitting `server.dcmjs.org` which is using dcm4chee, and so if I don't turn off `retrieveRendered`, nothing shows up.

Reply:
You are right. My bad. All good.